chart/k8gb/values.yaml

```
@@ -11,6 +11,7 @@ k8gb:
   # but can be overrided with imageTag key
   # imageTag:
   dnsZone: "cloud.example.com" # dnsZone controlled by gslb
+  dnsZoneNegTTL: 3600 # Negative TTL for SOA record
```

why default negative is 3600 when it is clearly problematic got GSLB scenario? 
```suggestion
  dnsZoneNegTTL: 60 # Negative TTL for SOA record
```

---

What about we don't set any default value and keep plugins one, if set it will be propogated

---

@k0da i'm concerned that another customers will hit the same issue without having a clue that the root cause is actually negative TTL.  It is safer to keep it low imho

---

10 minutes?

---

why?

---

IMO 1 minute is too agressive

---

agreed on 5 minutes